proto/reaperconfig.proto

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package proto;
+option go_package = "github.com/googleinterns/cloudai-gcp-test-resource-reaper";
+
+/*
+A reaper cluster is for running multiple reapers within the same GKE cluster,
+allowing for ease of organization and monitoring of all running reapers.
+*/
+message ReaperCluster {
+    // List of reapers that are running in the GKE cluster
+    repeated ReaperConfig configs = 1;
+}
+
+/*
+A reaper config describes all the resources the reaper will monitor, and how
+often the reaper should run. Note that any resource that matches the skip
+filter will be excluded, even if it matches a name filter in a resource config.
+*/
+message ReaperConfig {
+    // List of resources to watch
+    repeated ResourceConfig resources = 1;
+
+    // Frequency to run reaper in cron time string format
+    string schedule = 2;
+    
+    // Regex of names of resources to exclude
```

Comment:
In both cases where skipFilter is used please add an inline comment that skips win over matches

Reply:
Added